src/main/kotlin/racingcar/ResultView.kt

```diff
@@ -1,14 +1,17 @@
 package racingcar
 
-class ResultView {
-    companion object {
-        fun print(cars: List<Car>) {
-            for (car in cars) {
-                val location = car.currentLocation()
-                repeat(location) { print("-") }
-                println()
-            }
+object ResultView {
+    fun printLocation(cars: List<Car>) {
+        for (car in cars) {
+            val location = car.location
+            print(car.name + " : ")
+            repeat(location) { print("-") }
             println()
```

Comment:
아래와 같이 바꿔볼 수 있을 것 같아요. 😃 
```suggestion
            println("${car.name} : ${"-".repeat(car.location)}")
```